tests/helpers_spec.js

```diff
@@ -275,12 +275,33 @@ describe('Helpers', () => {
     });
   });
 
-  describe('privateKeyStringToKey', () =>
-    it('should convert sipa format', () => {
-      let res = Helpers.privateKeyStringToKey('5JFXNQvtFZSobCCRPxnTZiW1PDVnXvGBg5XeuUDoUCi8LRsV3gn', 'sipa');
+  describe('privateKeyStringToKey', () => {
+    let fixtures = [
+      { format: 'base58', key: 'DXuhJcNCPfgjYN8NQFjVh9yri8Rau5B7ixWJjpSeqW7W', addr: '1EgGs4Pd5ygy1ZCvMCgqpkrkMHtawYcnam' },
+      { format: 'base64', key: 'pbc8OCl5SdqjILx4R+yfvljZ7edUr65osjkzjqH2YQw=', addr: '1EL2ha3KWUFa2q7hfLk49XmUfiBV1yCzYG' },
+      { format: 'hex', key: '46DA1C47CF7FE23A97497665C217963F048EBC3E4287734947B180D64C7D81DF', addr: '19CK8suvcJptn96vfvLjfNnbcbEoyHS9G' },
+      { format: 'mini', key: 'S6c56bnXQiBjk9mqSYE7ykVQ7NzrRy', addr: '1PZuicD1ACRfBuKEgp2XaJhVvnwpeETDyn' },
+      { format: 'sipa', key: '5JFXNQvtFZSobCCRPxnTZiW1PDVnXvGBg5XeuUDoUCi8LRsV3gn', addr: '1BDSbDEechSue77wS44Jn2uDiFaQWom2dG' },
+      { format: 'compsipa', key: 'L3dDv2KUyLfPwkcnhALEaHnd47gewa1BVvnCwWL3gVWsb2H27PY9', addr: '1Mum7V38a3fGLiXN3oCrzdf83mZUKNXmGq' }
+    ];
+
+    fixtures.forEach(data => it(`should convert ${data.format} format`, () => {
+      let res = Helpers.privateKeyStringToKey(data.key, data.format);
       expect(Helpers.isKey(res)).toBeTruthy();
-    })
-  );
+      expect(res.getAddress()).toEqual(data.addr);
+    }));
+
+    it('should fail given a key with an invalid checksum', () => {
```

Comment:
We should do that over all the formats in in the fixtures, same as the positive check.